ViewDeck/IIViewDeckController.m

```diff
@@ -915,6 +915,38 @@ - (UIViewController *)childViewControllerForStatusBarStyle {
 
 #pragma mark - Appearance
 
+- (void)_forwardViewWillAppear:(BOOL)animated onViewController:(UIViewController *)viewController {
```

Comment:
I think Apple is pretty clear stating you shouldn't name methods with an underscore, see the "Private Methods" section of [Coding Guidelines for Cocoa](https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html).

> Don’t use the underscore character as a prefix for your private methods. Apple reserves this convention.

Perhaps would be better to name this something like `IIForwardViewWillAppear`.